src/app/common/common.tsx

```diff
@@ -199,7 +200,7 @@ class Tooltip extends React.Component<TooltipProps, TooltipState> {
         const style = this.calculatePosition();
 
         return ReactDOM.createPortal(
-            <div className="wave-tooltip" style={style}>
+            <div className={`wave-tooltip ${this.props.className}`} style={style}>
```

Comment:
change to: cn("wave-tooltip", this.props.className)
(better because it handles the undefined/null case automatically without an `|| ""`)